guides/v2.1/config-guide/varnish/config-varnish.md

```diff
@@ -52,21 +51,19 @@ The process can be summarized as follows:
 
 	If there is nothing in your `<your Magento install dir>/var/page_cache` directory, you've successfully configured Varnish with Magento!
 
-<div class="bs-callout bs-callout-info" id="info">
-	<ul><li>Except where noted, you must enter all commands discussed in this topic as a user with <code>root</code> privileges.</li>
-		<li>This topic is written for Varnish on CentOS and Apache 2.2. If you're setting up Varnish in a different environment, some commands are likely different. Consult Varnish documentation for more information.</li></ul>
-</div>
+{:.bs-callout .bs-callout-info #info}
```

Comment:
Make sure there is no blank line or hard return in this note, as it won't render properly.

Reply:
The note seems to be rendering correctly for me